src/components/Text/Text.test.js

```diff
@@ -0,0 +1,177 @@
+import React from 'react';
+import { shallow, render } from 'enzyme';
+import Text, { isParagraph, isHeading, getHeadingSize, selectTag } from './Text';
+
+describe('Checking if type is paragraph', () => {
+  it('for paragraph', () => {
+    expect(isParagraph('paragraph-1'))
+      .toBe(true);
+    expect(isParagraph('paragraph-2'))
+      .toBe(true);
+  });
+
+  it('for not paragraph', () => {
+    expect(isParagraph('heading-1'))
+      .toBe(false);
+    expect(isParagraph('paragrap-1'))
+      .toBe(false);
+  });
+
+  it('for wrong format paragraph', () => {
+    expect(isParagraph(' paragraph '))
+      .toBe(false);
+    expect(isParagraph(' paragraph-1'))
+      .toBe(false);
+    expect(isParagraph('paragraph-1 '))
+      .toBe(false);
+    expect(isParagraph('PARAGRAPH-2'))
+      .toBe(false);
```

Comment:
For multiple assertions like simple input => output (for functions) consider use https://www.npmjs.com/package/jest-each :)